packages/jest-editor-support/src/Snapshot.js

```diff
@@ -0,0 +1,156 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+const fs = require('fs');
+const walker = require('./walker');
+const utils = require('jest-snapshot/build/utils');
```

Comment:
can you make it so jest-snapshot exports the utils in index.js? Packages in Jest aren't allowed to be accessed through /build/ etc.